src/platform-apps/channels/channel-view.test.tsx

```diff
@@ -29,15 +39,35 @@ describe('ChannelView', () => {
     return shallow(<ChannelView {...allProps} />);
   };
 
-  it('renders channel name', () => {
+  const subjectWithContext = (props: any = {}, context: any = {}) => {
+    const allProps = {
+      name: '',
+      messages: [],
+      user: null,
+      countNewMessages: 0,
+      ...props,
+    };
+
+    const store = createStore(reducer);
+
+    return mount(
+      <Provider store={store}>
+        <AuthenticationContext.Provider value={{ ...context }}>
```

Comment:
are we just adding this provider so that we can test that it shows/hides the connect button? since we've tested the internal logic elsewhere, it should be sufficient here to just test that we are wrapping the message input with the correct IfAuthenticated component.